src/content/1.7/modules/creation/displaying_content_in_front_office.md

```diff
@@ -260,26 +268,33 @@ which is a must when using the translation tool.
 
 Here are our two files:
 
-    display.php
-    <?php
-    class mymoduledisplayModuleFrontController extends ModuleFrontController
+- *display.php*
+
+```php
+class mymoduledisplayModuleFrontController extends ModuleFrontController
+{
+    public function initContent()
     {
-      public function initContent()
-      {
         parent::initContent();
         $this->setTemplate('module:mymodule/views/templates/front/display.tpl');
-      }
     }
+}
+```
 
-    display.tpl
-    Welcome to my shop!
+- **display.tpl*
+```
+Welcome to my shop!
+```
 
 Let's explore `display.php`, our first PrestaShop front-end controller,
 stored in the /controllers/front folder of the module's main folder:
 
-    A front-end controller must be a class that extends the ModuleFrontController class.
-    That controller must have one method: `initContent()`, which calls the parent class' `initContent()` method...
-    ...which then calls the `setTemplate()` method with our `display.tpl` file.
+
+```
+A front-end controller must be a class that extends the ModuleFrontController class.
+That controller must have one method: `initContent()`, which calls the parent class' `initContent()` method...
+...which then calls the `setTemplate()` method with our `display.tpl` file.
+```
```

Comment:
I'm not sure this part should be in a code div

Reply:
Not sure too, but just formatting already existed.